Library/Formula/mkcramfs.rb

```diff
@@ -0,0 +1,36 @@
+class Mkcramfs < Formula
+  homepage "http://cramfs.sourceforge.net/"
+  url "https://downloads.sourceforge.net/sourceforge/cramfs/cramfs-1.1.tar.gz"
+  sha1 "08133f2a2a0f02c6dd07c8dc565a4eac6bc967cd"
+
+  def install
+    system "make", "mkcramfs"
+    bin.install "mkcramfs"
+  end
+
+  test do
+    system "#{bin}/mkcramfs", "-h"
+  end
+
+  patch :DATA
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!